xbmc/addons/addoninfo/AddonInfo.h

```diff
@@ -26,6 +26,18 @@ class CAddonInfo;
 typedef std::shared_ptr<CAddonInfo> AddonInfoPtr;
 typedef std::vector<AddonInfoPtr> AddonInfos;
 
+enum AddonDisabledReason
+{
+  // Special reason for returning all disabled addons.
+  // Only used as an actaul value when an addon is enabled.
+  ADDON_DISABLED_REASON_NONE = 0,
+  ADDON_DISABLED_REASON_UNKNOWN = 1,
```

Comment:
Not sure if the `ADDON_DISABLED_REASON_UNKNOWN` needed, as it can also maybe done by `ADDON_DISABLED_REASON_NONE` and not seen a place where used.